lib/components/map/base-layers.js

```diff
@@ -8,16 +9,23 @@ class BaseLayers extends Component {
     const { BaseLayer, Overlay } = LayersControl
     return (
       <LayersControl>
-        {baseLayers && baseLayers.map((l, i) => (
-          <BaseLayer
-            name={l.name}
-            checked={i === 0}
-            key={i}>
-            <TileLayer
-              {...l}
-              detectRetina />
-          </BaseLayer>
-        ))}
+        {baseLayers && baseLayers.map((l, i) => {
```

Comment:
Please rename `l` to `layers` since we also have this `L` variable. Maybe even rename `L` to `Leaflet`.

Reply:
Actually, is this file even needed anymore? It doesn't appear to be used anywhere. If not we should delete it.